src/viewer-selector.js

```diff
@@ -27,13 +27,18 @@ var CLASS_NAME = 'webvr-polyfill-viewer-selector';
  * saving the currently selected index in localStorage.
  */
 function ViewerSelector() {
-  // Try to load the selected key from local storage. If none exists, use the
-  // default key.
+  // Try to load the selected key from local storage.
   try {
-    this.selectedKey = localStorage.getItem(VIEWER_KEY) || DEFAULT_VIEWER;
+    this.selectedKey = localStorage.getItem(VIEWER_KEY);
   } catch (error) {
     console.error('Failed to load viewer profile: %s', error);
   }
+  
+  //If none exists, or if localstorage is unavailable, use the default key.
```

Comment:
good call; and [this too is handled](https://github.com/googlevr/webvr-polyfill/blob/c29bcbe42323bdd9411809b8272d9a942c4033e6/src/viewer-selector.js#L88) fine?